api/tests/core/users/email/test_update.py

```diff
@@ -2,6 +2,7 @@
 import datetime
 from unittest.mock import patch
 
+from freezegun.api import freeze_time
```

Comment:
(micro-détail) La fonction `freeze_time()` est accessible plus directement : 

```suggestion
from freezegun import freeze_time
```